eclair-core/src/test/scala/fr/acinq/eclair/channel/states/e/NormalStateSpec.scala

```diff
@@ -1803,6 +1803,23 @@ class NormalStateSpec extends TestKitBaseClass with FixtureAnyFunSuiteLike with
     awaitCond(alice.stateName == NORMAL)
   }
 
+  test("recv CMD_CLOSE (with unsigned fee update)") { f =>
```

Comment:
Have you considered adding a test for the first commit?

Reply:
Done in https://github.com/ACINQ/eclair/pull/1661/commits/d49e2f7cc295bb8bd991900c8d2203fe499c173b